app/models/tweet.rb

```diff
@@ -12,4 +12,9 @@ def generate_hashtags
       self.hashtags << hashtag
     end
   end
+
+  def self.search(search)
```

Comment:
This is way to slow. Especially since we have hashtags to search by!

Reply:
Also done.

Reply:
I think? Not totally sure about Ruby syntax.